install.sh

```diff
@@ -36,7 +36,7 @@ main() {
 
     ## Find and/or install the necessary tools
 
-    if [ "$(id -u)" = 0 ] || [ "$os" = Darwin ]; then
+    if am_admin; then
```

Comment:
I don't think we ever want to install using sudo on macos

Reply:
That doesn't seem right, but I guess I have never really used an account that isn't in the admin group. I think it's likely very uncommon